README.md

```diff
@@ -17,20 +17,20 @@ Dolphin Scheduler Official Website
 
 ### Design features:
 
-A distributed and easy-to-extend visual DAG workflow scheduling system. It dedicates to solving the complex dependencies in data processing to make the scheduling system `out of the box` for data processing.
+Dolphin Scheduler is a distributed and easy-to-extend visual DAG workflow scheduling system. It dedicates to solving the complex dependencies in data processing to make the scheduling system `out of the box` for the data processing process.
 
 Its main objectives are as follows:
 
- - Associate the tasks according to the dependencies of the tasks in a DAG graph, which can visualize the running state of task in real time.
- - Support many task types: Shell, MR, Spark, SQL (mysql, postgresql, hive, sparksql), Python, Sub_Process, Procedure, etc.
+ - Associate the tasks according to the dependencies of the tasks in a DAG graph, which can visualize the running state of the task in real-time.
+ - Support many task types: Shell, MR, Spark, SQL (MySQL, PostgreSQL, hive, spark SQL), Python, Sub_Process, Procedure, etc.
  - Support process scheduling, dependency scheduling, manual scheduling, manual pause/stop/recovery, support for failed retry/alarm, recovery from specified nodes, Kill task, etc.
- - Support process priority, task priority and task failover and task timeout alarm/failure
+ - Support process priority, task priority and task failover, and task timeout alarm/failure
```

Comment:
I hold the view that no need to use double 'and' here.
 - Support process priority, task priority, task failover, and task timeout alarm or failure
or
 - Support process priority, task priority, failover, and timeout alarm or failure
maybe better and please consider using 'workflow' instead of 'process'